certified-connectors/PartnerLinq/Readme.md

```diff
@@ -0,0 +1,62 @@
+## PartnerLinq Connector
```

Comment:
@FarhanJamilVSI this is a great readme file, but please try to include all required sections from our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md), and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/Independent%20Publisher/readme.md).